app/src/main/java/io/intrepid/contest/screens/contestcreation/NewContestContract.java

```diff
@@ -0,0 +1,34 @@
+package io.intrepid.contest.screens.contestcreation;
+
+import io.intrepid.contest.base.BaseContract;
+import io.intrepid.contest.base.TextValidatableView;
+import io.intrepid.contest.models.Contest;
+
+class NewContestContract {
+
+    public interface View extends BaseContract.View {
+
+        void onBackNavigationPressed();
+
+        void onForwardNavigationPressed();
```

Comment:
Yes, good point. i think onbackNavigationPressed() is only defined and called in my NewContestActivity, it should be a private method in the activity.